packages/flutter/test/widgets/scroll_aware_image_provider_test.dart

```diff
@@ -311,3 +363,20 @@ class RecordingPhysics extends ScrollPhysics {
     return super.recommendDeferredLoading(velocity, metrics, context);
   }
 }
+
+// ignore: must_be_immutable
```

Comment:
nit: please add comment why this is ignored.

Reply:
Done and done.  thanks for the quick review.